exercises/concept/magician-in-training/.docs/instructions.md

```diff
@@ -1,93 +1,94 @@
 # Instructions
 
-As a magician-to-be, Elyse needs to practice some basics. She has a stack of cards that she wants to manipulate.
+As a magician-to-be, Elyse needs to practice some basics.
+She has a stack of cards that she wants to manipulate.
 
-To make things a bit easier she only uses the cards 1 to 10.
+To make things a bit easier she only uses the cards 1 to 10 so her stack of cards can be represented by an array of numbers.
```

Comment:
Why not using Strings. Each card can be represented by a String? What's the difference?

Reply:
It is just what the instructions say to make it easier, as you might notice this is not a new implementation of the exercise (I didn't write the exercise), more of a rewrite of the text. The idea is that the exercise stay the same.